pkg/dhtnode-common.pkg

```diff
@@ -5,18 +5,14 @@ OPTS.update(
     description = FUN.desc(OPTS, 'common files'),
     architecture = 'all',
     depends = 'dhtnode',
-    deb_upstart = [
-        'deploy/upstart/dht.conf',
-        'deploy/upstart/dhtnode-instance.conf',
-        'deploy/upstart/dhtdump.conf',
-        'deploy/upstart/dhtdump-instance.conf',
-    ],
 )
 
 ARGS = FUN.mapfiles('deploy/logrotate', '/etc/logrotate.d',
         ('dhtnode-logs', 'dhtdump-logs'), append_suffix=False) + \
     [
         "README.rst=/usr/share/doc/" + VAR.shortname + "/README.rst",
+        "deploy/systemd/dht@.service=/lib/systemd/system/dht@.service",
+        "deploy/systemd/dhtdump@.service=/lib/systemd/system/dhtdump@.service",
```

Comment:
Did you look into `--deb-systemd FILEPATH        (deb only) Add FILEPATH as a systemd script` ?

Reply:
Ah [I see why](https://github.com/jordansissel/fpm/issues/1363).
Would be nice to add a comment... And fix the bug ? Because that'll allow to keep both upstart and systemd script IIUC.

Reply:
:-)